src/index.js

```diff
@@ -10,17 +10,37 @@ const token = require('./helpers/jwt');
 
 const GITHUB_TOKEN = process.env.GITHUB_TOKEN;
 const JWT_SECRET = process.env.JWT_SECRET;
+const WEBHOOK_SECRET = process.env.WEBHOOK_SECRET;
+const DEBUG = process.env.DEBUG;
 
 const returnErrorResponse = message => Promise.resolve(http.response(message, http.STATUS_CODE.ERROR));
 
 async function lambda(event) {
+  if (DEBUG) console.log(event);
+
   if (!GITHUB_TOKEN) { return returnErrorResponse(config.responseMessages.githubTokenNotProvided); }
   if (!JWT_SECRET) { return returnErrorResponse(config.responseMessages.jwtTokenNotProvided); }
   if (!event) { return returnErrorResponse(config.responseMessages.lambdaEventObjectNotFound); }
 
   let requestBody;
   let requestParams = {};
 
+  /**
+   * It seems like there are two types of Event object headers naming conventions.
```

Comment:
Was this identified through a blog/research/docs, or by trial and error?
Can be beneficial to know where this knowledge came from (in the case of it getting updated for example)

Reply:
Trial and error based on the previous version of code as well as internet source confirmations. Headers get normalized right now in the code to lower-case so we should be good whenever anything changes in the future. There is multiple scenarios where it could change including custom overwrites defined on API Gateway.